server.lua

```diff
@@ -4,61 +4,73 @@
 -- Forwards any data POSTed to /events to an event-stream at /events.
 -- Serves files from /static otherwise.
 
-local turbo = require('turbo')
-
-local subscribers = {}
-
-function postEvent(data)
-  for k, v in pairs(subscribers) do
-    k:sendEvent(data)
-  end
-end
-
-local EventHandler = class('EventHandler', turbo.web.RequestHandler)
-
-function EventHandler:sendEvent(data)
-  self:write('data: ')
-  self:write(data)
-  self:write('\n\n')
-  coroutine.yield(self:flush())
-end
-
-function EventHandler:get()
-  self:set_async(true)
-  self:set_chunked_write()
-  self:set_header('Content-Type', 'text/event-stream')
-  self:set_header('Cache-Control', 'no-cache')
-  self:set_header('Connection', 'keep-alive')
-  self:set_status(200)
-  self:flush()
+local port = tonumber(arg[1]) or 8000
+local hostname = arg[2] or '127.0.0.1'
 
-  subscribers[self] = self
-end
+local async = require('async')
```

Comment:
if you wish, i wrote the same file, completely devoid of async. It is ugly and non-performant in the sense that it is synchronous and the rudimentary http parsing is done in Lua. Would that be acceptable-ish (I hate it tbh!)? if not, we can have your server as a first preference, and have a fallback to the async based server.